src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Runtime/BasicTestForEmptyStringsUsingStringLength.Fixer.vb

```diff
@@ -34,5 +34,24 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Runtime
         Protected Overrides Function GetRightOperand(binaryExpressionSyntax As SyntaxNode) As SyntaxNode
             Return DirectCast(binaryExpressionSyntax, BinaryExpressionSyntax).Right
         End Function
+
+        Protected Overrides Function IsFixableBinaryExpression(node As SyntaxNode) As Boolean
+            Return (TypeOf node Is BinaryExpressionSyntax) AndAlso (IsEqualsOperator(node) Or IsNotEqualsOperator(node))
+        End Function
+
+        Protected Overrides Function IsFixableInvocationExpression(node As SyntaxNode) As Boolean
+            Return node.IsKind(SyntaxKind.InvocationExpression)
+        End Function
+
+        Protected Overrides Function GetInvocationTarget(node As SyntaxNode) As SyntaxNode
+            If (TypeOf node Is InvocationExpressionSyntax) Then
```

Comment:
Instead of a TypeOf check followed by a DirectCast, you can just use a TryCast and check for non-null

Reply:
Is there a specific reason to do this? Best practices? Performance?

Reply:
Yes, performance - you are replacing two operations with one